### DISCUSSION TITLE
Option to add additional comments in the generated ORM files.

### DISCUSSION THREAD

**Discussion by nohhyun** [aarondl/sqlboiler#1405] [Ideas]
I am wondering if people would be interested in having an option to add a specific string to the top of the generated file. 
I am thinking something like this can be added for the `sqlboiler.toml`. 
```
output   = "models"             
pkgname  = "models"             
wipe     = true                
no-tests = true
add-enum-types = true  
additional-top-comment = "@generated"
```

Which will generate files with 
```
// Code generated by SQLBoiler (https://github.com/volatiletech/sqlboiler). DO NOT EDIT.
// This file is meant to be re-generated in place and/or deleted at any time.
// @generated
```
at the top.
I can generate a quick pull request if this seems acceptable.

**Comment by stephenafamo** [comment]
PR welcome 👍🏾
